great_expectations/datasource/fluent/pandas_datasource.py

```diff
@@ -153,7 +152,7 @@ def get_batch_list_from_batch_request(self, batch_request: BatchRequest) -> list
             batch_request=batch_request, ignore_options=("dataframe",)
         )
 
-        batch_list.append(
+        return [
```

Comment:
This function only ever returns one batch, so updated the method setup to reflect that.